src/operator/roi_pooling.cc

```diff
@@ -74,7 +79,13 @@ inline void ROIPoolForward(const Tensor<cpu, 4, Dtype> &out,
 
     const Dtype* batch_data = bottom_data + data_size * roi_batch_ind;
 
+    #pragma omp parallel for firstprivate(batch_data, top_data, argmax_data)
```

Comment:
curious why it’s not “parallel for” pragma?

Reply:
Do you mean why there is a `firstprivate` clause here?

Reply:
Oh, I totally missed the 'for' in the pragma. Sorry.